### PR TITLE
MSSM bbH/bbA powheg cards

### PR DESCRIPTION
**Pull request by Tommaso93**
Dear GEN conveners,
these are the b-associated powheg cards in the MSSM HiggsToMuMu analysis with UL pdfs.

Best,
Tommaso

### CODE REVIEW

**Comment by Tommaso93** [comment]
reminder

File: bin/Powheg/production/2017/13TeV/Higgs/MSSM/bbH_MuMu_NNPDF31_13TeV/powheg.input_template [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+manyseeds  0       ! Used to perform multiple runs with different random
+                   ! seeds in the same directory.
+                   ! If set to 1, the program asks for an integer j;
+                   ! The file pwgseeds.dat at line j is read, and the
+                   ! integer at line j is used to initialize the random
+                   ! sequence for the generation of the event.
+                   ! The event file is called pwgevents-'j'.lhe
+
+storeinfo_rwgt 1   ! store info for reweighting events
```
Comment (agrohsje) [inline bin/Powheg/production/2017/13TeV/Higgs/MSSM/bbH_MuMu_NNPDF31_13TeV/powheg.input_template:84]: as in the ggH request.

**Comment by agrohsje** [comment]
I would suggest to clean up a bit the cards, like obsolete pdf's. I see you posted a problem on the HN about failing higher masses. Let me add @covarell  to the thread to comment on the limitations for higher mass points:
https://hypernews.cern.ch/HyperNews/CMS/get/generators/5066.html

File: bin/Powheg/production/2017/13TeV/Higgs/MSSM/bbH_MuMu_NNPDF31_13TeV/powheg.input_template (hunk 1 above)
Comment (Tommaso93) [reply bin/Powheg/production/2017/13TeV/Higgs/MSSM/bbH_MuMu_NNPDF31_13TeV/powheg.input_template:84]: Ok!

**Comment by Tommaso93** [comment]
Thank you! Concerning the HN thread, is quite a strange behaviour! For each mass point, e.g. 1000GeV, I have multiple gridpacks, each one with a different tanb (5,10,15,...). Two of them managed to finish quickly, the others are stuck (as I explained in the mail).

**Comment by Tommaso93** [comment]
Dear @agrohsje, thank you for the help! The problem of the condor jobs stucked was solved. I also cleaned the cards, as requested (here and in the ggH thread)

**Comment by Tommaso93** [comment]
If everything is ok, can you please close the PR?

**Comment by agrohsje** [comment]
Hi @Tommaso93 , it seems something went wrong, e.g. I see this file twice in the commit: 
2017/13TeV/Higgs/MSSM/bbh_MuMu_NNPDF31_13TeV/powheg.input_template

**Comment by Tommaso93** [comment]
They are two different files, bbH and bbh. It is the same card with only one parameter different

**Comment by Tommaso93** [comment]
@agrohsje Should I close and reopen the PR? From my side, I see the correct files to be merged (both here and #2915 )

**Comment by agrohsje** [comment]
Hi @Tommaso93 . I looked at the line for quite a bit but didn't spot H vs h.

**Comment by Tommaso93** [comment]
Hi @agrohsje, sorry for the confusion. The card is the same as bbH, but the mass value and tanb in the makecard are different (computed with a specific mssm tool).

**Comment by Tommaso93** [comment]
@agrohsje I decided to put the same file for clarity in the folder structure.

**Comment by agrohsje** [comment]
looks ok to me. let me add @covarell for a second check.

**Comment by Tommaso93** [comment]
Thanks @agrohsje

**Comment by agrohsje** [comment]
@covarell just pinged me. Going to merge.

**Pull request merged**